roles/openshift_logging/templates/elasticsearch.yml.j2

```diff
@@ -28,11 +28,10 @@ cloud:
 discovery:
   type: kubernetes
   zen.ping.multicast.enabled: false
-  zen.minimum_master_nodes: {{es_min_masters}}
+  zen.minimum_master_nodes: {{es_node_quorum}}
```

Comment:
@ewolinetz I presume we should maybe convert this to rely on the env variable?  or modify line 34 to use the variable.  Thoughts?  Should these all use vars given they are unlikely to change during the deployment? @portante thoughts?